src/codeu/chat/server/Controller.java

```diff
@@ -143,6 +143,30 @@ public ConversationHeader newConversation(Uuid id, String title, Uuid owner, Tim
     return conversation;
   }
 
+  public void newConvoInterest(Uuid user, Uuid idToSave, int numberOfMessageOfConvo)
+  {
+    final User foundUser = model.userById().first(user);
+    foundUser.addConvoInterest(idToSave,numberOfMessageOfConvo);
```

Comment:
Thanks for pointing that out! Currently, I'm just pulling from a value that Matt calculates, but I'll definitely look into that.